Source/WebCore/bindings/scripts/CodeGeneratorJS.pm

```diff
@@ -5018,6 +5029,29 @@ sub GenerateImplementation
         push(@implContent, "}\n\n");
     }
 
+    if ($interfaceName eq "NodeList") {
```

Comment:
Would be nice to move this to `JSNodeListCustom.cpp`.

Reply:
Hmm.. moved to `JSNodeListCustom.cpp`. But got compile error shown in https://ews-build.webkit.org/#/builders/2/builds/22559. Any idea how to resolve that?

Reply:
I think this is simply some headers are missing. Let's see the error message and add missing headers.